### DISCUSSION TITLE
time

### DISCUSSION THREAD

**Discussion by acehero8** [jojo2357/Music-Discord-Rich-Presence#46] [Ideas]
hey i love your drpc tool its awesome
can you put time remain/left timer to the rpc 
thanks!

**Comment by jojo2357** [comment]
In 1.6.0, this is planned for musicbee, but unfortunately it is not possible at this time with groove because I cannot use a plugin or anything to get the time left/elapsed but with musicbee I am making a plugin to get that information.

**Reply by acehero8** [reply]
thanks man! you really doing great job 👍  cant wait for the next project for bee player
cheers
